components/src/preact/components/error-display.tsx

```diff
@@ -25,6 +25,7 @@ export class UserFacingError extends Error {
 }
 
 export const ErrorDisplay: FunctionComponent<{ error: Error }> = ({ error }) => {
+    // eslint-disable-next-line no-console -- Currently this is our error handling, we need to show it somewhere
```

Comment:
Was a bit confused by this comment as I wasn't sure if it is a TODO or a mere statement of the current error handling set-up. In case it is the later maybe we could change the lines to:
```suggestion
    // eslint-disable-next-line no-console -- Currently we use the following statement for our error handling
```